index.js

```diff
@@ -1,17 +1,60 @@
 class SortedList {
-  constructor() {}
+  constructor(items, length) {
+    this.items = [],
+    this.length =this.items.length
+  }
 
-  add(item) {}
+  add(item) {
+    const list = this.items.push(item)
+    list.sort(function(a, b) {
+      return a - b
+    })
+    this.length = list.length
+    return list
```

Comment:
No te haria falta devolver la lista, de hecho no tienes que sacarlo a una constante, porque si la quieres ordenar, si ordenamos this.items pues cuando instanciamos una variable const x = new SortedList ya tendriamos dentro de x esos items no?
```suggestion
```